diff --git a/.config/doom/config.org b/.config/doom/config.org
index 99fa539..f806d88 100644
--- a/.config/doom/config.org
+++ b/.config/doom/config.org
@@ -49,7 +49,7 @@ I am still undecided on usage of variable pitch font. I cannot recall what issue
 *** Theme
 
 #+begin_src elisp
-(setq doom-theme 'doom-tokyo-night)
+(setq doom-theme 'doom-one-light)
 #+end_src
 
 Who doesn't want Bruce Lee to greet them at the beginning of a work session. He's there to remind you:
