diff --git a/.config/gitui/theme.ron b/.config/gitui/theme.ron
index bd80cec..79617e2 100644
--- a/.config/gitui/theme.ron
+++ b/.config/gitui/theme.ron
@@ -1,8 +1,8 @@
 (
     selected_tab: Reset,
-    command_fg: Gray,
-    selection_bg: Rgb(11,41,66),
-    cmdbar_extra_lines_bg: Rgb(11,41,66),
+    command_fg: White,
+    selection_bg: LightBlue,
+    cmdbar_extra_lines_bg: LightBlue,
     disabled_fg: DarkGray,
     diff_line_add: Green,
     diff_line_delete: Red,
